docs/gremlint/package.json

```diff
@@ -11,8 +11,10 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.53",
     "@types/react-dom": "^16.9.8",
-    "gremlint": "github:OyvindSabo/gremlint#master",
+    "customize-cra": "^1.0.0",
+    "gremlint": "^3.5.1",
```

Comment:
I set the gremlint version to `^3.5.1` for now, as opposed to `^3.5.1-alpha1` or `^3.5.1-rc1` as I wasn't sure which convention we were going with and also not sure if we would actually want the website to use the alpha versions.:thinking:

Reply:
the only part in my mind where it gets tricky is with the release process, but i think you have it right. when we release 3.5.2 we dont' publish docs until after we'd have deployed that artifact to npm at which point we'd get the newer version to be live on the website at gremlint.com.